pkg/manifests/nginx_test.go

```diff
@@ -139,6 +139,33 @@ var (
 				TargetCPUUtilizationPercentage: 80,
 			},
 		},
+		{
+			Name: "internal-with-ssl-cert-and-force-ssl",
+			Conf: &config.Config{
+				NS:          "test-namespace",
+				Registry:    "test-registry",
+				MSIClientID: "test-msi-client-id",
+				TenantID:    "test-tenant-id",
+				Cloud:       "test-cloud",
+				Location:    "test-location",
+			},
+			Deploy: &appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-operator-deploy",
+					UID:  "test-operator-deploy-uid",
+				},
+			},
+			IngConfig: &NginxIngressConfig{
+				ControllerClass:                "test-controller-class",
+				ResourceName:                   "nginx",
+				IcName:                         "nginx-private",
+				DefaultSSLCertificate:          "fakenamespace/fakename",
+				ForceSSLRedirect:               true,
+				MinReplicas:                    2,
+				MaxReplicas:                    100,
+				TargetCPUUtilizationPercentage: 80,
+			},
+		},
```

Comment:
add test case with forceSSL redirect true but no default ssl cert to prove it doesn't add the forcessl